spec/features/user_show_spec.rb

```diff
@@ -0,0 +1,72 @@
+require 'spec_helper'
+
+RSpec.describe 'User integrations', type: :feature do
+  it 'displays user profile in show page' do
+    user = create(:user)
+    visit user_path(user)
+    expect(page).to have_content('Andrew Carnegie')
+  end
+
+  it 'Can see user name ' do
+    user = create(:user)
+    visit user_path(user)
+    expect(page).to have_content('Andrew Carnegie')
+  end
+  it 'displays total number of posts' do
+    user = create(:user)
+    create(:post, user:)
+    create(:post, user:)
+    visit user_path(user)
+    expect(page).to have_content('Posts Number')
+  end
+  it 'displays user bio' do
+    user = create(:user)
+    visit user_path(user)
+    expect(page).to have_content('Bio')
+  end
```

Comment:
- [ ] [REQUIRED] This test creates a user, visits the user's show page, and then checks if the page contains the string 'Bio'. While this test verifies that there is some content related to the user's bio on the page, it doesn't specifically check if the user's actual bio is displayed.

Here is an example of the actual test expected:
``` ruby
it 'displays user bio' do
  user = create(:user, bio: 'Lorem ipsum dolor sit amet')
  visit user_path(user)
  expect(page).to have_content(user.bio)
end
```